### PR TITLE
wiiscrubber: Add version 1.4

### PR DESCRIPTION
**Pull request by UnicorNora**
[x ] I have read the [Contributing Guide](../CONTRIBUTING.md).

## What package release type is this?
- [x ] stable
- [ ] beta
- [ ] dev
- [ ] nightly/canary

- [ ] properly named and capitalized shortcuts?
- [ ] autoupdate and checkver entry [](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate)
- [ ] [persist](https://github.com/ScoopInstaller/Scoop/wiki/Persistent-data) defined with config/data/user/portable/textures/saves folder(s) specific for the app?
- [ ] a [pre_install](https://github.com/ScoopInstaller/Scoop/wiki/Pre--and-Post-install-scripts) script to auto-enable portable mode (if needed)?
  - [ ] script pre-creates generated files listed in persist? (solves linking issue with generated files that need to be persisted)
- [ ] license identifier and url
- [ ] a short terse description matching existing style.
- [ ] url to the release along with its sha256 hash
- [ ] bin entries
    - [ ] if beta, dev, nightly, canary, etc: does bin have rewrite with variant appended to the name?
- [ ] I tested the package and it runs in portable mode. I have verified the symlinks are correct in the persists folder.
- [ ] I will maintain this manifest and promptly fix it in the event it breaks.
- [ ] passes `bin/checkver.ps1` and `bin/checkurls.ps1`
- [ ] tested install, checked persist, no errors.

### CODE REVIEW

**Comment by UnicorNora** [comment]
sorry im really tired today

File: bucket/wiiscrubber.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": "1.4",
+    "description": "An app that allows you to 'scrub' away the garbage data of Wii ISO.",
```
Comment (Calinou) [inline bucket/wiiscrubber.json:3]:
```suggestion
    "description": "Tool to 'scrub' away the garbage data of Wii ISOs",

```

File: bucket/wiiscrubber.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.4",
+    "description": "Tool to 'scrub' away the garbage data of Wii ISOs",
+
```
Comment (Calinou) [inline bucket/wiiscrubber.json:4]:
```suggestion
    "description": "Tool to 'scrub' away the garbage data of Wii ISOs",

```

File: bucket/wiiscrubber.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.4",
+    "description": "Tool to 'scrub' away the garbage data of Wii ISOs",
+
+    "homepage": "https://gamebanana.com/tools/6467",
```
Comment (Calinou) [inline bucket/wiiscrubber.json:5]:
```suggestion
    "homepage": "https://gamebanana.com/tools/6467",

```

File: bucket/wiiscrubber.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.4",
+    "description": "Tool to 'scrub' away the garbage data of Wii ISOs",
+    "homepage": "https://gamebanana.com/tools/6467",
+
```
Comment (Calinou) [inline bucket/wiiscrubber.json:5]:
```suggestion
    "homepage": "https://gamebanana.com/tools/6467",
```

File: bucket/wiiscrubber.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        "script": [
+            "Expand-7zipArchive \"$dir\\Wiiscrubber140.rar\" \"$dir\"",
+            "Expand-7zipArchive \"$dir\\MakeKeyBin1.1.rar\" \"$dir\"",
+            "Expand-7zipArchive \"$dir\\WiiPartitionBinChanger.rar\" \"$dir\\WiiPartitionBinChanger\"",
+            "Expand-7zipArchive \"$dir\\Multiboot.rar\" \"$dir\\Multiboot\"",
+            "Expand-7zipArchive \"$dir\\partitionbuilder.rar\" \"$dir\"",
+            "Remove-Item \"$dir\\`*.rar\", \"$dir\\*.rar\" -Force -Recurse"
+        ]
+    }
+}
```
Comment (Calinou) [inline bucket/wiiscrubber.json:33]:
```suggestion
    "version": "1.4",
    "description": "Tool to 'scrub' away the garbage data of Wii ISOs",
    "homepage": "https://gamebanana.com/tools/6467",
    "license": "Freeware",
    "url": "https://gamebanana.com/dl/385923#/dl.rar",
    "hash": "f1af4a22ead44af0870378bc11915c4b90d4ae986a3bd3a821edb9f9c28aa5f4",
    "bin": [
        "WiiScrubber.exe",
        "MakeKeyBin.exe"
    ],
    "shortcuts": [
        [
            "WiiScrubber.exe",
            "Wii Scrubber"
        ],
        [
            "MakeKeyBin.exe",
            "Wii Scrubber - Make Common Key"
        ]
    ],
    "installer": {
        "script": [
            "Expand-7zipArchive \"$dir\\Wiiscrubber140.rar\" \"$dir\"",
            "Expand-7zipArchive \"$dir\\MakeKeyBin1.1.rar\" \"$dir\"",
            "Expand-7zipArchive \"$dir\\WiiPartitionBinChanger.rar\" \"$dir\\WiiPartitionBinChanger\"",
            "Expand-7zipArchive \"$dir\\Multiboot.rar\" \"$dir\\Multiboot\"",
            "Expand-7zipArchive \"$dir\\partitionbuilder.rar\" \"$dir\"",
            "Remove-Item \"$dir\\`*.rar\", \"$dir\\*.rar\" -Force -Recurse"
        ]
    }
}

```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**